<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>753</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OCR Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Images/icon.png</normaloff>:/Images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="LB_MainView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>711</width>
        <height>441</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>691</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="BTN_ImageOpen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Open image</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="BTN_CamOpen">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Open camera</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="BTN_VideoOpen">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Open video</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="BTNPause">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTabWidget" name="Algorithms">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>251</width>
         <height>271</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Detection</string>
        </attribute>
        <widget class="QCheckBox" name="CB_TextDetectorEnable">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Enable cascade detector</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
        <widget class="QLabel" name="LB_ScoreTresholdVal">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Score threshold</string>
         </property>
        </widget>
        <widget class="QSlider" name="SB_ScoreTreshold">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>82</y>
           <width>181</width>
           <height>18</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Adjust scale factor</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="LB_NMSThresholdVal">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>130</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="SB_NMSThreshold">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>143</y>
           <width>181</width>
           <height>18</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Adjust min neigbor</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>120</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NMS threshold</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Recognition</string>
        </attribute>
        <widget class="QCheckBox" name="CB_TextRecogEnable">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BTN_WriteText2File">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>90</y>
           <width>111</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QComboBox" name="CBB_WriteMethod">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>111</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Overwrite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Append</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="PTE_AppLog">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>251</width>
         <height>231</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="PTE_TextOutput">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
